python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/asset_decorator.py

```diff
@@ -12,6 +12,9 @@
 from dagster_embedded_elt.sling.dagster_sling_translator import DagsterSlingTranslator
 from dagster_embedded_elt.sling.sling_replication import SlingReplicationParam, validate_replication
 
+METADATA_KEY_TRANSLATOR = "dagster_sling/translator"
+METADATA_KEY_REPLICATION_CONFIG = "dagster_sling/replication_config"
```

Comment:
We should hide this metadata from the UI: https://github.com/dagster-io/dagster/pull/20493

Reply:
Oh! Had no idea this existed, thanks for pointing it out. Added here: https://github.com/dagster-io/dagster/pull/20564/commits/cb8f22cfe6b75b160ecc35353e46b299318f510b